libhoney/client.py

```diff
@@ -84,9 +91,6 @@ def __init__(self, writekey="", dataset="", sample_rate=1,
         if debug:
             self._init_logger()
 
-        def IsClassicKey(key):
```

Comment:
It has been a hot minute since I did Python seriously, and I can't remember if this method ended up being public this way or not? If it _is_ in fact public, I'll re-add it to call the above to avoid the breaking change